amusement-park/amusement-park.js

```diff
@@ -0,0 +1,77 @@
+/// <reference path="./global.d.ts" />
+
+import { visitors } from "@babel/traverse";
+import { version } from "os";
+
+// @ts-check
+
+/**
+ * Creates a new visitor.
+ *
+ * @param {string} name
+ * @param {number} age
+ * @param {string} ticketId
+ * @returns {Visitor} the visitor that was created
+ */
+export function createVisitor(name, age, ticketId) {
```

Comment:
You can attach the properties to the object directly while initializing it.

Reply:
@JuanVimon3 can you please take a look at this suggestion and update the file accordingly?